testsuite/features/step_definitions/common_steps.rb

```diff
@@ -251,10 +250,10 @@
 
 # systemspage
 Given(/^I am on the Systems page$/) do
-  steps %(
```

Comment:
Not sure this will work, I think it needs specific syntax

Reply:
Nope.
step and steps are regular ruby methods imported from the Cucumber library.
It does not need any special syntax, as far as we don't use special characters on the string passed by parameter.

Reply:
So, to give an example, using %(...) you don't have to escape the " character inside the string.
While if you use "" or '' you will need to escape some characters.

Reply:
All good then